src/util/devices.tsx

```diff
@@ -62,3 +65,24 @@ export const deviceKeyToLabel = (input: string): string => {
   }
   return input;
 };
+
+export const getExistingDeviceNames = (
+  values: InstanceAndProfileFormValues,
+  profiles: LxdProfile[],
+): string[] => {
+  const existingDeviceNames: string[] = [];
+
+  // Returns devices within the instance
+  existingDeviceNames.push(...values.devices.map((item) => item.name));
+  // Returns devices within the profile
```

Comment:
```suggestion
  // Returns devices from any of the profiles assigned to the instance
```

Reply:
Thank you! I have addressed this now, I'll just need your approval again :).